activation/licensed-users-count.md

```diff
@@ -0,0 +1,34 @@
+---
+description: This article explains how SysKit Point licensing depends on the Microsoft 365 licenses in your tenant.
+---
+
+# Licensed Users Count
+
+The price of the SysKit Point subscription **depends on the number of licensed users within the Microsoft 365 tenant connected to SysKit Point**.
+
+**Licensed Users** are all users with any type of Microsoft 365 license attached to them.
+ * **Guest Users** only fit into that category **if they have a license assigned to them**. 
+   * Having a license assigned to Guest Users is unnecessary for rudimentary use cases such as sharing content and using the Microsoft Teams app.
+
+The number of licensed Users within Syskit Point can be found by going to **Settings (1) > General (2) > Point License (3) > Number of Users M/N (4)**.
+
+![Inactive Licensed Users](../.gitbook/assets/activation-licensed-users-point.png)
```

Comment:
tekst koji je u uglatim zagradama, u ovo slučaju [Inactive Licensed Users] je vidljiv u članku kada se publisha, prikazan je ispod slike. treba kratko opisati što slika prikazuje. 
ovdje bi to bilo Number of Licensed Users